### PR TITLE
Updating help center link

### PR DESCRIPTION
**Pull request by eoji**
# what
Updating help center link!
It follows the same pattern as our other help links.

# why
@walterbm said so

# see how cute

### CODE REVIEW

File: app/src/main/java/com/kickstarter/ui/activities/HelpActivity.java [hunk 1]
```diff
@@ -106,6 +113,9 @@ protected String getUrlForHelpType(final @HelpType int helpType) {
       case HELP_TYPE_FAQ:
```
Comment (Scollaco) [inline app/src/main/java/com/kickstarter/ui/activities/HelpActivity.java]: I don't think we need HELP_TYPE_FAQ anymore.

File: app/src/main/java/com/kickstarter/ui/activities/HelpActivity.java [hunk 2]
```diff
@@ -30,8 +30,9 @@
   public static final int HELP_TYPE_HOW_IT_WORKS = 2;
   public static final int HELP_TYPE_COOKIE_POLICY = 3;
   public static final int HELP_TYPE_FAQ = 4;
```
Comment (Scollaco) [inline app/src/main/java/com/kickstarter/ui/activities/HelpActivity.java]: HELP_TYPE_HELP_CENTER is replacing HELP_TYPE_FAQ, so we wouldn't need both here.

**Comment by eoji** [comment]
@nnekab @Scollaco @walterbm  Made those changes.

Here's the updated gif:

**Comment by eoji** [comment]
[BUG: We link to an outdated Help Center via app settings](https://trello.com/c/PxKJYDBt/740-bug-we-link-to-an-outdated-help-center-via-app-settings)

**Comment by eoji** [comment]
made more changes due to an issue with the webview ༼ ༎ຶ ෴ ༎ຶ༽ We are now linking ✨externally ✨in both settings and the Login/Sign up helper menu

cc @walterbm

**Pull request merged**